services/base/repository.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: source.toolkit.fluxcd.io/v1beta2
 kind: GitRepository
 metadata:
   name: services-repository
-  namespace: config-services-flux
+  namespace: config-cluster-flux
 spec:
   interval: 5m
   ref:
-    branch: main
+    branch: features/kb/flux-changes
```

Comment:
@kbabigumira : should we change it to Main when other PR get completed?

Reply:
yes